### ISSUE TITLE
Please stop using this Crappy old repo

### ISSUE THREAD

**Issue by ghost** [arut/nginx-dav-ext-module#74]
You should not use this repo, the software isn't fully implemented (e.g. PROPPATCH is missing), and no you can use it with Windows or OS X to mount it as a drive. Especially due to the missing WebDAV methods and the even more ugly implementation on Windows and OS X (as most of their software is awful). Their webdav clients are awful... Only Filezilla Pro works fine for the moment

**Comment by konomikitten** [comment]
This is probably the best advice I've found on this repository. The code hasn't been touched in 6 years. Nginx does not play well with WebDAV at all. I've spent a good week on and off experimenting with Nginx and WebDAV and it has been extremely frustrating to say the least.

Don't use Nginx for WebDAV,  just don't, the default module in Nginx itself is too strict with the trailing slash issue. You will find websites with hacks to try and work around this but they'll fall over and break in certain conditions. I'm also pretty sure I'm hitting race conditions with WebDAV as well, but it's not like I can report this in any meaningful way when this repository is abandoned.

TL;DR: Don't expect WebDAV with Nginx to work properly.

**Comment by biergaizi** [comment]
Right now, this module is 100% legacy code and no longer supports Nginx 1.24+. Most systems have already completed removed this module from the package (e.g. https://cgit.freebsd.org/ports/commit/www/nginx/Makefile?id=364b314d88996ec0e94b44e8b4297bb52170e4cb )
